react/README.md

```diff
@@ -57,6 +57,29 @@ To see an example of the code running, follow these steps:
 
 The code of the example is in [`react/example`](https://github.com/text-mask/text-mask/tree/master/react/example).
 
+## Customize Rendered `<input>` Component
+
+For advanced uses, you can customize the rendering of the resultant `<input>` via render prop. This
```

Comment:
What do you think of changing 
> the resultant `<input>` via render prop.

to read 

> the resultant `<input>` via **a** render prop.

Reply:
Hey, thanks for taking a look!  Agreed, just updated this.